src/Hangfire.Mongo/packages.config

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Hangfire.Core" version="1.5.3" targetFramework="net451" />
-  <package id="MongoDB.Bson" version="2.1.1" targetFramework="net451" />
-  <package id="MongoDB.Driver" version="2.1.1" targetFramework="net451" />
-  <package id="MongoDB.Driver.Core" version="2.1.1" targetFramework="net451" />
+  <package id="MongoDB.Bson" version="2.2.0" targetFramework="net451" />
+  <package id="MongoDB.Driver" version="2.2.0" targetFramework="net451" />

```

Comment:
As a note, as this has been upgraded to the 2.2 driver, All of the AsyncHelper code should probably be removed as their are now synchronous calls that mirror all of the async calls in the 2.2 drop of the Mongo driver.  Most everywhere where you have .ToListAsync()  you can just replace it with .ToList()